### PR TITLE
Add matplotlib-base back as a required dependency

### PR DESCRIPTION
**Pull request by xylar**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
Might as well get a proper review from you this time, @ocefpaf and @dopplershift

**Review by ocefpaf — APPROVED** [review]
LGTM. Thanks @xylar!

File: recipe/meta.yaml
Comment (dopplershift) [inline recipe/meta.yaml:38]: Typo "matpltolib"

**Review by dopplershift — APPROVED** [review]
Good to go after fixing typo

File: recipe/meta.yaml
Comment (ocefpaf) [reply recipe/meta.yaml:38]: I'm pretty sure that is correct. Isn't that the library used to plot ptolemy's model of the universe?

Comment (xylar) [reply recipe/meta.yaml:38]:
```suggestion
    # including matplotlib-base even though it's optional upstream
```
Aargh!  I can't type today.  Thanks @dopplershift

**Pull request merged**
